schematic/src/gschem_about_dialog.c

```diff
@@ -65,23 +65,48 @@ void about_dialog (GschemToplevel *w_current)
     g_error_free (error);
   }
 
-  gtk_show_about_dialog (
-      GTK_WINDOW (w_current->main_window),
-      "version",        version_string,
-      "logo",           logo,
-      "title",          _("About gschem"),
-      "comments",       _("gEDA: GPL Electronic Design Automation"),
-      "copyright",
-      /* TRANSLATORS: "ChangeLog" is a literal filename; please don't translate it. */
```

Comment:
Please don't remove this line for translators!

Reply:
Sorry about that! I'll resurrect it.